### ISSUE TITLE
Experimental futex2 interface doesn't work

### ISSUE THREAD

**Issue by buzzcut-s** [ValveSoftware/wine#103]
futex2 with latest experimental hangs at startup and is unusable. This is because at https://github.com/ValveSoftware/wine/blob/experimental_5.13/server/fsync.c#L86
```        
if (use_futex2 && (f = fopen( "/sys/kernel/futex2/wait", "r" )))
        {
            char buffer[13];

            fgets( buffer, sizeof(buffer), f );
            nr_futex2_wake = atoi( buffer );
            fclose(f);

            do_fsync_cached = 1;
        }
```

Should be `f = fopen( "/sys/kernel/futex2/wake", "r" )` instead, since we're assigning to `nr_futex2_wake`.

**Comment by phush0** [comment]
no game is starting at all just say "fsync start" and that's it logs are empty, i am with futex2 patched kernel. Even adding `WINEFSYNC_FUTEX2=0` is not good. 

EDIT: Patch is good. With it all games are playable again.

**Comment by aeikum** [comment]
PR was merged and included in the experimental-5.13-20210125 build. Thanks.
